concepts/05 UI Components/Toolbar/00 Getting Started with Toolbar/00 Getting Started with Toolbar.md

```diff
@@ -0,0 +1,22 @@
+#include tutorials-intro-installationnote
+
+The Toolbar contains items that manage the page content.
+
+This tutorial shows how to add the Toolbar component to your application and configure its core features.
+
+<div class="simulator-desktop-container" data-view="/Content/Applications/21_2/GettingStartedWith/Toolbar/index.html, /Content/Applications/21_2/GettingStartedWith/Toolbar/index.js, /Content/Applications/21_2/GettingStartedWith/Toolbar/index.css"></div>
+
+Each section in this tutorial describes a single configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-devextreme-toolbar" target="_blank">getting-started-with-toolbar</a>.
+
+The Toolbar consists of four sections: left area, central area, right area, and overflow menu. Follow the list below to learn how to add items to the Toolbar:
```

Comment:
You create a table of contents (at least it looks like a table of contents). But it's missing one item then - Customize Items. Maybe it's better to include it to avoid confusion... Or maybe you rely on some side navigation? 

Just to be consistent, I guess you can change the introduction like this: 

```suggestion
You can add items to the toolbar in its four sections: left area, central area, right area, and overflow menu. Templates help you customize items displayed in any of these section. Refer to the following tutorial steps for details:
```

And then you can add another item about templates (Customize Items).